rust/src/config/cli.rs

```diff
@@ -116,6 +117,10 @@ impl Cli {
                     return Err(format!("Unknown arguments: '{}'.", m.free.join(" ")));
                 }
 
+                if m.opt_str("datadir").is_some() {
+                    warn!("The `datadir` command line argument is deprecated, use `gamedir` instead");
+                }
+
```

Comment:
is this intentional? Appears to be for/from the PR about the argument renaming.

Reply:
Yes, I just wanted to have an easy usecase for logging in Rust. And those were not logged yet.